### PR TITLE
hotfix/v0.13.5

### LINKED ISSUE #82: Change all python scripts to executable
The following scripts should be made executable (`chmod +x`):
1. `add_grouper_groups`
2. `get_org_code_numbers`
3. ~`grouper_permission_checks`~ (not part of v0.13.4)
4. `script_run`
5. `user_update`

This can be implemented as `hotfix/v0.13.5`

Also, updates to [README.md](README.md) for simpler execution.

Although outside this issue, for deployment aliases on patron server needs to be modified. This is not crucial but helps to keep things clean.

### PR DESCRIPTION
**Pull request by damian-romero**
- [x] Change executing permissions to python scripts inside `scripts/`
- [x] Add Python shebang to `get_org_code_numbers`
- [x] Update README.md to reflect changes and bump version

Fixes #82

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,7 +279,8 @@ We use [SemVer](http://semver.org/) for versioning. For the versions available,
 A list of released features and their issue number(s).
 List is sorted from moderate to minor revisions for reach release.
 
-v0.13.0 - v0.13.4:
+v0.13.0 - v0.13.5:
+ * All scripts under the `scripts/` are now executable
```
Comment (astrochun) [inline README.md]:
```suggestion
 * Make scripts executable #82 
```

File: README.md [hunk 2]
```diff
@@ -255,7 +258,7 @@ GrouperSuperAdmins and GrouperAdmins. If a group already exists, it will
 skip to the privilege assignments.  To execute the script:
 
 ```
-(figshare_patrons) $ python scripts/add_grouper_groups --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/add_grouper_groups --config config/figshare.ini \
```
Comment (astrochun) [inline README.md]:
```suggestion
(figshare_patrons) $ ./scripts/add_grouper_groups --config config/figshare.ini \
```

File: README.md [hunk 3]
```diff
@@ -199,7 +202,7 @@ To this end, the `user_update` script should be used. It has several features:
 Execution can be done as follows:
 
 ```
-(figshare_patrons) $ python scripts/user_update --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/user_update --config config/figshare.ini \
```
Comment (astrochun) [inline README.md]:
```suggestion
(figshare_patrons) $ ./scripts/user_update --config config/figshare.ini \
```

File: README.md [hunk 4]
```diff
@@ -156,7 +159,7 @@ To execute the script and update Grouper and EDS, include the `portal`, `quota`,
 and `sync` command-line flags:
 
 ```
-(figshare_patrons) $ python scripts/script_run --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --config config/figshare.ini \
```
Comment (astrochun) [inline README.md]:
```suggestion
(figshare_patrons) $ ./scripts/script_run --config config/figshare.ini \
```

File: README.md [hunk 5]
```diff
@@ -145,7 +148,7 @@ Without the `sync` flag, the above command line will perform a
 To undo this change, use the `test_reverse` flag:
 
 ```
-(figshare_patrons) $ python scripts/script_run --test_reverse \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --test_reverse \
```
Comment (astrochun) [inline README.md]:
```suggestion
(figshare_patrons) $ ./scripts/script_run --test_reverse \
```

File: README.md [hunk 6]
```diff
@@ -131,7 +134,7 @@ Test command-line flags (`test` and `test_reverse`) are available to test EDS
 query and Grouper synchronization (with the `sync` flag) by executing the following :
 
 ```
-(figshare_patrons) $ python scripts/script_run --test \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --test \
```
Comment (astrochun) [inline README.md]:
```suggestion
(figshare_patrons) $ ./scripts/script_run --test \
```

File: README.md [hunk 7]
```diff
@@ -122,7 +125,7 @@ execute the following command:
 ```
 (figshare_patrons) $ export password="insert_password"
 (figshare_patrons) $ export persist_path="/path/to/persistent/storage"
-(figshare_patrons) $ python scripts/script_run --config config/figshare.ini \
+(figshare_patrons) $ ./ReQUIAM/scripts/script_run --config config/figshare.ini \
```
Comment (astrochun) [inline README.md]:
```suggestion
(figshare_patrons) $ ./scripts/script_run --config config/figshare.ini \
```

**Review by astrochun — COMMENTED** [review]
Changes look good.  A few things.

1. I've simplified to call since ReQUIAM is the default path (./ReQUIAM/scripts -> ./scripts)
2. I'm not sure what happened but the Table of Contents was changed substantially and it should not have been. See below for what it looks like.  There are some indentation issues:

Screen Shot 2020-11-10 at 1 41 56 PM

2. Please bump version to `v0.13.5` in `README.md`, `setup.py`, and `requiam/__init__.py`

**Comment by damian-romero** [comment]
Good to go after I solved the `README.md` TOC issue.

**Review by astrochun — APPROVED** [review]
Perfect.

Rebased some of your commits

**Pull request merged**